### PR TITLE
navigation button added to testimonial

### PR DESCRIPTION
**Pull request by Ankita297**
## What is the change?
navigation button added to testimonial
## Related issue?
close: #777 

## How was it tested?
I have tested it in my local system 
## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [x] Did you lint your code before making the Pull Request?
- [x] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video:
Add a screenshot or demo video if appropriate.

https://user-images.githubusercontent.com/67833077/116803367-88668000-ab34-11eb-9c38-b001ac4588db.mp4

### CODE REVIEW

**Comment by Ankita297** [comment]
@Pranav016  Please review my PR .

**Comment by Pranav016** [comment]
> @Pranav016 Please review my PR .

The left right buttons work right ?

**Comment by Ankita297** [comment]
Yes ,they work .

**Comment by Pranav016** [comment]
> And did you check if the feature is responsive for mobile view ?

@Ankita297

File: views/partials/testimonial.ejs [hunk 1]
```diff
@@ -18,31 +18,41 @@
...
-            <a href="/testimonial-write"><button class="button pen test-button">Write a testimonial</button></a>
+            <div class="text-center">
+                <a href="/testimonial-write"><button class="button pen test-button">Write a testimonial</button></a>
+            </div>
     </div>
-</div> 
- 
\ No newline at end of file
+</div>
+</div>
```
Comment (Pranav016) [inline views/partials/testimonial.ejs]: this code might need some formatting to be done manually, fix the indentations

**Review by Pranav016 — CHANGES_REQUESTED** [review]
One change otherwise looks good to merge

File: views/partials/testimonial.ejs (hunk 1 above)
Comment (Ankita297) [reply views/partials/testimonial.ejs]: @Pranav016  what is indentations and how can i resolve it ?
and for responsiveness i have check it done bit changes  and it is responsive if you can tell how to  fix the indentations then i will make the commit .

**Comment by Ankita297** [comment]
WordNook - Google Chrome 02-05-2021 17_08_34

**Comment by Ankita297** [comment]
@Pranav016 please see it .

**Comment by Ankita297** [comment]
@Pranav016 what about indentations ?

File: views/partials/testimonial.ejs (hunk 1 above)
Comment (Pranav016) [reply views/partials/testimonial.ejs]: With indentation I mean basic code formatting in hierarchy like- (example)
```
<div>
<a href="..">
</div>
```
when correctly indented
```
<div>
    <a href="..">
</div>
``` 
or 
```
                                        <% } %>
                                </div>
                                            <% } %>
                                    </div>
```
this when correctly formatted-
```
                                        <% } %>
                                </div>
                         <% } %>
                  </div>
```

Comment (Ankita297) [reply views/partials/testimonial.ejs]: Okay

**Comment by Ankita297** [comment]
@Pranav016 Please see now .

**Review by Pranav016 — APPROVED** [review]

**Pull request merged**